other/web-js/client/src/components/AppName.js

```diff
@@ -6,6 +6,7 @@ const API_TYPE = process.env.REACT_APP_API_TYPE;
 const API_TYPE_NAMES = {
   graphql: "GraphQL",
   mql: "MQL",
+  "data-api": "Data API",
   local: "Local",
 };
```

Comment:
idea: rather than having this here + additional metadata in `other/web-js/client/src/components/MoreInfo.js`, maybe create a `metadata.js` file with API type names, docs link, and any other API type-specific config